tests/emscripten_runner.js

```diff
@@ -86,7 +86,7 @@ async function main() {
     FS.mkdir('/test_dir');
     FS.mount(FS.filesystems.NODEFS, {root: path.join(root_dir, 'tests')}, '/test_dir');
     FS.chdir('/test_dir');
-    await pyodide.loadPackage(['micropip', 'pytest']);
+    await pyodide.loadPackage(['micropip', 'pytest', 'pygments']);
```

Comment:
Is this related / intentional?

Reply:
it's required since `inline-snapshots` requires it.